interface/frontend/src/components/report-and-export/RSNAStandardTemplate.vue

```diff
@@ -213,7 +216,27 @@ export default {
   props: [ 'rsna' ],
   methods: {
     exportRSNA () {
-
+      // specified the column with all the 3 components in the view
+      const pdfSection = $('#pdf-section')
+      html2canvas(pdfSection).then(canvas => {
+        var imgData = canvas.toDataURL('image/png')
```

Comment:
Not sure, but think it'll be better to follow the @louisgv suggestion to use `let` and `const` in lue of `var`